app/styles/headers/_fifth.scss

```diff
@@ -0,0 +1,161 @@
+.header__fifth {
+    background: url(../img/header5/background.jpg) 100% no-repeat;
+    background-size: cover;
+    font-family: $font-face;
+    font-size: 14px;
+    color: white;
+    position: relative;
+    margin-top: 60px;
+    margin-bottom: 60px;
+
+}
+
+.fifth {
+    width: 66.1%; // 780px / 1180px - don't use static width for adaptive
+    margin: 0 auto;
+    padding-top: 80px;
+
+    &__menu {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__navigation {
+        width: 57%; // 445px / 780px - don't use static width for adaptive
+        display: flex;
+        justify-content: space-between;
+
+        a {
+            text-transform: uppercase;
+            color: inherit;
+            text-decoration: none;
+            font-weight: 600;
+            letter-spacing: 0.05em; // 50(in layout) / 1000 - convert data from psd to css
+        }
+    }
+
+    &__title, &__subsection {
+        width: 60.3% // 470px / 780px - don't use static width for adaptive
+    }
+
+    &__title {
+        font-size: 50px;
+        font-weight: 600;
+        letter-spacing: 0.02em;
+        margin-top: 60px;
+        line-height: 64px;
+    }
+
+    &__subsection {
+        font-size: 18px;
+        letter-spacing: 0.02em;
+        margin-top: 30px;
+        line-height: 24px;
+        font-weight: 600;
+    }
+    
+    &__btn {
+        background-color: #d9c26c;
+        border: none;
+        color: inherit;
+        font-family: inherit;
+        font-size: 18px;
+        outline: none;
+        margin-top: 45px;
+        padding: 22px 54px; // data from psd
+        border-radius: 40px;
+        letter-spacing: 0.02em;
+        line-height: 24px;
+        font-weight: 600;
+        margin-bottom: 110px;
+    }
+
+    &__touches {
+        display: inline-block;
+        position: absolute;
+        user-select: none;
+        bottom: 45px;
+        left: calc(50% - 98px / 2); // 98px - .fifth__touches width 
+    }
+
+    &__touch {
+        display: inline-block;
+        margin: 0 4px;
+        border: solid 3px white;
+        border-radius: 50%;
+        cursor: pointer;
+        width: 8px;
+        height: 8px;
+
+        &:hover {
+            border-color: darken(white, 20%);
+        }
+
+        &--active {
+            background-color: white;
+        }
+    }
+
+    &__arrow {
+        width: 40px;
+        height: 40px;
+        border: solid 3px white;
+        border-radius: 50%;
+        color: white;
+        font-size: 25px;
+        text-align: center;
+        position: absolute;
+        cursor: pointer;
+        background-color: rgba(0, 0, 0, 0);
+        top: calc(50% - 23px); // 23px - half of .fifth__arrow's height
+
+        i {
+            position: absolute;
```

Comment:
try to use `padding: 4px` (or find required value with testing different values) in `&__arrow` to align icon in the center

Reply:
`padding: 4px ` is the same thing with magic number, like I do it.
I don't know how to center it without that.